tests/utils/test_tunic.py

```diff
@@ -25,6 +25,35 @@ def test_get_active_campaigns(self):
                               2: {'id': 2,
                                   'name': 'Another Campaign'}})
 
+    def test_get_all_paginated_campaigns(self):
+        with requests_mock.mock() as mocker:
```

Comment:
I was actually writing something like 

```
class MockResponse(object):

    def __init__(self, content):
        self.content = content

    def json(self):
        return self.content```

on the onion until I saw `requests_mock` in bulbs :(